src/main/java/org/folio/rest/impl/LoginAPI.java

```diff
@@ -180,133 +167,139 @@ private String getTenant(Map<String, String> headers) {
         requestURL = String.format("%s/users?query=id==%s", okapiURL,
             URLEncoder.encode(userId, "UTF-8"));
       }
-    } catch(Exception e) {
-      logger.error("Error building request URL: " + e.getLocalizedMessage());
-      future.fail(e);
-      return future;
+    } catch (UnsupportedEncodingException e) {
+      // Should never happen - UTF-8 is supported
```

Comment:
Using https://github.com/folio-org/raml-module-builder/blob/v30.0.0/util/src/main/java/org/folio/util/StringUtil.java avoids this try-catch.